openapi/index.yaml

```diff
@@ -338,25 +338,204 @@ paths:
+  "/services":
+    post:
+      operationId: createService
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: |-
+            A service can invoke the IO API by providing the relative API key
+            (every service has an associated primary and secondary API key).
+            Through the API a service can send messages to the IO users
+            that haven't opted out from it. Service metadata are used
+            to qualify the sender of these messages to the recipient.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service created.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+        "500":
+          description: The service cannot be created.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    get:
+      operationId: getUserServices
+      summary: Get User Services
+      description: |
+        Retrieve the identifiers of the services owned by the calling user
+      responses:
+        "200":
+          description: The list of service ids.
+          schema:
+            "$ref": "#/definitions/ServiceIdCollection"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+  "/services/{service_id}":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of an existing Service.
+    get:
+      operationId: getService
+      summary: Get Service
+      description: |
+        Retrieve a previously created service with the provided service ID.
+        This API operation needs the same API key of the service being retrieved
+        otherwise 403 forbidden will be returned to the caller.
+      responses:
+        "200":
+          description: Service found.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
+          description: Unauthorized.
+        "403":
+          description: Forbidden.
+        "404":
+          description: No service found for the provided ID.
+        "429":
+          description: Too many requests.
+    put:
+      operationId: updateService
+      summary: Update Service
+      description: Update a previously created service with the provided service ID
+      parameters:
+        - in: body
+          name: body
+          description: |-
+            The updated service payload.
+            This API operation needs the same API key of the service being updated
```

Comment:
I think this is related to the operation and not to the parameter.

Reply:
Fixed in [469e173](https://github.com/pagopa/io-functions-services/pull/67/commits/469e173201722102ed04c60b05300f6618a02fe2)